Makefile

```diff
@@ -1,24 +1,11 @@
-obj-m := parrot.o
-parrot-objs := src/ffi.o src/rust.o
-
-all: src/rust.o
-	$(call if_changed,make -C /lib/modules/$(shell uname -r)/build M=$(PWD) modules)
-
-.PHONY:
-fmt:
-	cargo fmt
-
-.PHONY:
-clippy:
-	cargo clippy
+all: 
+	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) modules
 
 clean:
-	cargo clean
-	rm -rf rust_objs
 	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) clean
 
-src/rust.o: Cargo.toml src/ffi.c src/lib.rs src/macros.rs src/mutex.rs src/parrot.rs
-	cargo rustc --release -Z build-std=core,alloc --target=x86_64-unknown-none-linuxkernel
-	mkdir -p rust_objs
-	cd rust_objs && ar x ../target/x86_64-unknown-none-linuxkernel/release/librust.a
-	ld -r -o src/rust.o rust_objs/*.o
+fmt-check:
```

Comment:
It would be great using the same names for the targets as in-tree, i.e. `rustfmt` and `rustfmtcheck` to be consistent.

Maybe I will add them to the template example...